templates/watcher/config/httpd.conf

```diff
@@ -32,6 +33,7 @@ AccessFileName .htaccess
         LogLevel debug
         EnableSendfile On
 
+        TypesConfig /etc/mime.types
```

Comment:
so previously we were getting mime.conf becuase of the Include on the next line 37...
but you now removed that, so we need to configure this TypesConfig here explicitly ?
(for my own understanding)

Reply:
exactly, that's what other operators do, e.g nova https://github.com/openstack-k8s-operators/nova-operator/blob/0933c978bad435a91ec3fff1121f762a93e9bba9/templates/novaapi/config/httpd.conf#L13 or keystone https://github.com/openstack-k8s-operators/keystone-operator/blob/ced167413cf979dfdbaec16468115894af26384c/templates/keystoneapi/config/httpd.conf#L13